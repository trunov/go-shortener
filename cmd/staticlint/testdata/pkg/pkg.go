package pkg

import "os"

func main() {
	os.Exit(1)
}
