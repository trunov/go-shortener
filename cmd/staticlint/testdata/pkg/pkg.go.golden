analysistest.go:498: /Users/trunov/Desktop/main/projects/go-shortener/cmd/staticlint/testdata/pkg/pkg.go:6:2: unexpected diagnostic: direct call to os.Exit found in main function